### DISCUSSION TITLE
Cannot verify my student status

### DISCUSSION THREAD

**Discussion by KhaledElwazan** [community/community#27360] [Copilot Conversations]
I'm a student at Faculty of science, Alexandria University. I was submitting my faculty ID as a proof of my academic status to obtain a student access to copilot. I tried to submit my faculty ID on the discount request form on github, but it was rejected because the name of the faculty does not appear on the same side as my photo ID and affiliation info, but it resides on the back of my ID. I tried to upload both ID faces in a single image using Gimp, but it was rejected as well.

Any thoughts?

**Comment by sawyerbivens** [comment]
I suggest you make a ticket with the Education support team [(here)](https://support.github.com/contact/education)

It may take some time as I've seen some complaints on other discussions here regarding the education support team being slow.

Good luck though!

**Comment by Dread2** [comment]
I had the same issue, and just got verified. It was weird. After saying it could not verify my documentation, I went back to the original page where it asks for your email, resubmitted the documents and it worked. You could try that.

**Reply by vineet-codes256** [reply]
I'm facing the same issue. There is no Upload button, we have to take pictures with our laptop's camera in which it is impossible to get a clear image, there is no text detection  or auto-focus, it fails every time.

**Reply by davecheney** [reply]
@vineet-codes256 i'm sorry you are having difficulties. Sadly this is not the right forum to ask for support. Please contact support, https://support.github.com/contact/education
